tests/integration-tests/tests/schedulers/test_slurm/test_slurm_memory_based_scheduling/pcluster.config.mem-based-scheduling.yaml

```diff
@@ -27,7 +27,9 @@ Scheduling:
         - Name: ondemand1-i2
           Instances:
             - InstanceType: c5.xlarge
+            {% if "us-iso" not in region %}
```

Comment:
Here and below: why not replacing c5a.xlarge with c5d.xlarge for every region and avoid the conditional branch?

Reply:
There already is c5d.xlarge as an option. I could remove c5a.xlarge altogether for all the regions though i dont think that makes sense. as in one region we could have higher availability of those than the other two options we have mentioned

Reply:
Sorry i did not express my thoughts the right way. I mean replacing c5a.xlarge with an equivalent one in all regions, eg m5.xlarge